dts/binding-template.yaml

```diff
@@ -87,6 +87,7 @@ on-bus: <string describing bus type, e.g. "i2c">
 # A typical property entry looks like this:
 #
 #   <property name>:
+#     not: <true | false>
```

Comment:
I think:

    forbidden: <true | false>

would be more meaningful.  "not" works for JSON where the value is a type description, rather than an assertion of absence.  It also pairs naturally with "required".